### PR TITLE
bump @metamask/network-controller from v10.3.1 to v11.0.0

### PR DESCRIPTION
**Pull request by adonesky1**
Bumps `@metamask/network-controller` from v10.3.1 to v11.0.0 and adds migration to add ticker to current `providerConfig` state if its missing: [relevant v11.0.0 changelog entry](https://github.com/MetaMask/core/blob/main/packages/network-controller/CHANGELOG.md#changed-1)

### CODE REVIEW

File: app/scripts/migrations/092.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    const { providerConfig } = state.NetworkController;
+
+    if (providerConfig.ticker) {
+      return state;
+    }
+
+    state.NetworkController.providerConfig = {
+      ...providerConfig,
+      ticker: 'ETH',
+    };
```
Comment (jiexi) [inline app/scripts/migrations/092.ts:41]: should the spread come last? 

```
    state.NetworkController.providerConfig = {
      ticker: 'ETH',
      ...providerConfig,
    };
```

Comment (adonesky1) [reply app/scripts/migrations/092.ts:41]: yes whoopsss

Comment (adonesky1) [reply app/scripts/migrations/092.ts:41]: Though if a ticker is present and truthy this won't even hit, but still I'll push the change.

Comment (adonesky1) [reply app/scripts/migrations/092.ts:41]: done here: [0f1f838](https://github.com/MetaMask/metamask-extension/pull/20367/commits/0f1f838c02b4d7b9e040643417a4deab4643d823)

Comment (jiexi) [reply app/scripts/migrations/092.ts:41]: ahh i missed that guard somehow. I don't think the guard is needed if the spread comes last. No strong opinion though, feel free to leave as is

**Review by jiexi — APPROVED** [review]

**Review by BelfordZ — APPROVED** [review]

**Review by brad-decker — APPROVED** [review]

**Pull request merged**